apps/dolly-backend/src/main/java/no/nav/dolly/DollyBackendLifecycle.java

```diff
@@ -0,0 +1,50 @@
+package no.nav.dolly;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.dolly.repository.BestillingRepository;
+import org.springframework.context.SmartLifecycle;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Component
+@Profile({"dev", "local", "prod"})
+@RequiredArgsConstructor
+@Slf4j
+public class DollyBackendLifecycle implements SmartLifecycle {
+
+    private final BestillingRepository repository;
```

Comment:
Gjøre det samme for OrganisasjonBestillingRepository også? 
Opplever at de noen ganger henger over lengre tid